File: official/docs/java/current/referral-customers/add-payment-method-with-bank-account.java
package referral;

import com.easypost.exception.EasyPostException;
import com.easypost.model.PaymentMethodObject;
import com.easypost.service.EasyPostClient;

public class Create {
    public static void main(String[] args) throws EasyPostException {
        EasyPostClient client = new EasyPostClient("EASYPOST_API_KEY");

        PaymentMethodObject paymentMethod = client.betaReferralCustomer.addPaymentMethod("cus_...", "ba_...");

        System.out.println(paymentMethod);
    }
}

File: official/docs/golang/current/referral-customers/add-payment-method-with-bank-account.go
package example

import (
	"fmt"

	"github.com/EasyPost/easypost-go/v5"
)

func addPaymentMethodWithBankAccount() {
	client := easypost.New("EASYPOST_API_KEY")

	paymentMethod, _ := client.BetaAddPaymentMethod("cus_...", "ba_...", easypost.PrimaryPaymentMethodPriority)

	fmt.Println(paymentMethod)
}
